x-pack/plugins/apm/server/routes/create_apm_api.ts

```diff
@@ -52,13 +51,13 @@ import {
   correlationsForFailedTransactionsRoute,
 } from './correlations';
 import {
-  transactionGroupsBreakdownRoute,
-  transactionGroupsChartsRoute,
-  transactionGroupsDistributionRoute,
-  transactionGroupsRoute,
-  transactionSampleForGroupRoute,
-  transactionGroupsErrorRateRoute,
-} from './transaction_groups';
+  transactionsChartsBreakdownRoute,
```

Comment:
nit: Singular "transaction" sounds better

```suggestion
  transactionChartsBreakdownRoute,
```

Reply:
Same with the others below.